### PR TITLE
bigdl keras tutorial

### PR DESCRIPTION
**Pull request by megaSpoon**
inspired and modified from http://gluon.mxnet.io/chapter02_supervised-learning/linear-regression-scratch.html

### CODE REVIEW

File: .idea/BigDL-Tutorials.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
```
Comment (zhichao-li) [inline .idea/BigDL-Tutorials.iml]: remove?

File: .idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
```
Comment (zhichao-li) [inline .idea/modules.xml]: remove this file?

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
```
Comment (zhichao-li) [inline .idea/vcs.xml]: remove?

File: .idea/BigDL-Tutorials.iml (hunk 1 above)
Comment (megaSpoon) [reply .idea/BigDL-Tutorials.iml]: ye

File: .idea/modules.xml (hunk 2 above)
Comment (megaSpoon) [reply .idea/modules.xml]: removed

File: .idea/vcs.xml (hunk 3 above)
Comment (megaSpoon) [reply .idea/vcs.xml]: removed

**Comment by yiheng** [comment]
please rename the folder from bigdl-learning-mate to keras(Done)

**Comment by yiheng** [comment]
For ptb.train.txt file and other data file, can we download them from the internet instead of checking them to the repo.(Done)

**Comment by yiheng** [comment]
Please rename the PR title(Done)

**Comment by yiheng** [comment]
In Convolutional-Neural-Networks-With-BigDL-Keras.ipynb

1. Image credit: Stanford cs231n doesn't show (Done)
2. In Define The Model session, should we remove the commented code (Done)
3. What's the training result? Any curve, accuracy, prediction result?

**Comment by yiheng** [comment]
In Recurrent Neural Networks with BigDL
1. Should we put some result in the notebook(e.g. curve, accuracy or prediction result)

**Comment by yiheng** [comment]
Do we have an index for all these keras notebooks?

**Comment by yiheng** [comment]
Same question for the Linear-Regression-With-BigDL-Keras, no result. Is it a matter?

**Comment by yiheng** [comment]
Looks like the Logistic-Regression-Bigdl-Keras is unfinished

**Comment by megaSpoon** [comment]
Our current BigDL Keras API doesn't expose the loss information in the training. So the visualization of the loss curve is not plausible but the training accuracy and validation accuracy can be seen from the terminal where the notebook is started. Will use TensorBoard as another workround.

Saying "Do we have an index for all these keras notebooks?" Do you mean providing links to these notebooks in the README?

File: .gitignore [hunk 4]
```diff
@@ -69,6 +69,7 @@ target/
 
 # IPython Notebook
 .ipynb_checkpoints
+bigdl-learning-mate/*.ipynb_checkpoints
```
Comment (yiheng) [inline .gitignore:72]: bigdl-learning-mat -> keras?

**Comment by yiheng** [comment]
If user can see the accuracy in the terminal, I'm okay with it. But you should mention this(better with screenshot) in the notebooks.

**Comment by yiheng** [comment]
Exactly, add some links in the Readme

**Comment by yiheng** [comment]
There's an exception in the notebook https://github.com/megaSpoon/BigDL-Tutorials/blob/8aa9a28cf529470330e64a7e1194e12be8988742/keras/recurrent_neural_networks/Recurrent-Neural-Network-with-BigDL-Keras.ipynb

**Comment by yiheng** [comment]
This notebook is unfinished?
https://github.com/megaSpoon/BigDL-Tutorials/blob/8aa9a28cf529470330e64a7e1194e12be8988742/keras/supervised_learning/Logistic-Regression-Bigdl-Keras.ipynb

**Comment by megaSpoon** [comment]
fixed and ready to merge
